Traceback (most recent call last):
  File "QHE_notes_figures.py", line 304, in <module>
    make_new_data=mnd, pt=plot_type, plt_ss=True)
  File "QHE_notes_figures.py", line 255, in data_maker
    plot_ss=plt_ss)
  File "QHE_notes_figures.py", line 88, in named_plot_creator
    DATA = qt.mesolve(H_0, rho_0, timelist, [L_full], e_ops=expects, progress_bar=True, options=opts)
  File "/home/henry/anaconda2/envs/qutip/lib/python2.7/site-packages/qutip/mesolve.py", line 284, in mesolve
    progress_bar)
  File "/home/henry/anaconda2/envs/qutip/lib/python2.7/site-packages/qutip/mesolve.py", line 790, in _mesolve_const
    return _generic_ode_solve(r, rho0, tlist, e_ops, opt, progress_bar)
  File "/home/henry/anaconda2/envs/qutip/lib/python2.7/site-packages/qutip/mesolve.py", line 1020, in _generic_ode_solve
    r.y, 1)
KeyboardInterrupt

